.env.sample

```diff
@@ -0,0 +1,15 @@
+# .env.local
+
+# Select an experiment collection
+# Prod link
+RECORDS_URL="https://firefox.settings.services.mozilla.com/v1/buckets/main/collections/nimbus-desktop-experiments/records"
+
+# Preview link
+# RECORDS_URL="https://firefox.settings.services.mozilla.com/v1/buckets/main/collections/nimbus-preview/records"
+
+# Secrets
+EXPERIMENTER_API_KEY=
+NODE_ENV="development"
+
+# Dashboardy things
+FORCED_ENROLLMENT_PREFIX="optin-"
```

Comment:
We don't appear to be using this.  Do we need it?

Reply:
I added it thinking it would be the next thing I played around with. I'll keep it out of this patch for simplicity!